hyperspy/_signals/lazy.py

```diff
@@ -23,7 +23,8 @@
 
 import numpy as np
 import dask.array as da
-import dask
+import dask.threaded as dt
+from dask import __version__
```

Comment:
Is it good? This will add the `__version__` attribute to this module with its value set to the dask version!

Reply:
It avoids importing the whole of dask, just to have the version available. Any better idea?